components/MonitoringStatusOverviewTable.vue

```diff
@@ -76,7 +76,7 @@ export default Vue.extend({
   methods: {
     replaceFullWidthByHalfWidth(str: string) {
       return str.replace(
-        /[０-９]/g,
+        /[０-９|（|）]/g,
```

Comment:
もしかしたら以下のコードでもいけるかも？
```suggestion
        /[０-９（）]/g,
```

Reply:
いけますね